docs/docs-book/src/using-coffee.md

```diff
@@ -104,6 +104,14 @@ To install a plugin statically, you simply need to run.
 coffee install <plugin_name>
 ```
 
+#### Specifying a branch for installation
+
+To install a plugin from a specific branch, you simply need to run.
+
+```bash
+coffee install <plugin_name> --branch <branch_name>
+```
+
```

Comment:
we should not document every single command possibility otherwise our documentation will be always uncompleted

Reply:
Right. I can remove this section

Reply:
but we should came up with a way to document optional parameters